src/commands/test-branch.yml

```diff
@@ -11,64 +11,77 @@ parameters:
   command_verb:
     type: string
     default: Runs tests
+  rails_version:
+    type: string
+    default: ''
+  working_directory:
+    type: string
+    default: '~/project'
```

Comment:
This matches the default job `working_directory`. See https://circleci.com/docs/configuration-reference/#job-name.